views/js/buckaroo.js

```diff
@@ -194,6 +216,15 @@ function buckaroo($) {
                 case 'payperemail':
                     methodValidator.payPerEmailTrigger();
                     break;
+                case 'ideal':
+                    methodValidator.requiredIssuers('ideal_issuer','#booIdealErr','Please select an issuer.');
+                    break;
+                case 'paybybank':
+                    methodValidator.requiredIssuers('paybybank_issuer','#booPayByBankErr','Please select an issuer.');
+                    break;
+                case 'creditcard':
+                    methodValidator.requiredIssuers('creditcard_banks','#booCreditCardErr','Please choose your credit or debit card.');
```

Comment:
I think it will be better if we move the error message in the twig template there we can translate it, so the `requiredIssuers` function can only show the error message if the fields are not valid